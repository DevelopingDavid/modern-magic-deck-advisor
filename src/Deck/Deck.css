.deck--container {
  width: 216px;
  margin: 10px;
}

.deck--image {
  height: 300px;
  border-radius: 12px;
}

.counter--container {
  display: flex;
  justify-content: flex-end
}

.card--counter {
  margin: 5px 0;
  font-size: 1.5rem;
}