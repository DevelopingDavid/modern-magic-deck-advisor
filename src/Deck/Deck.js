import React, { Component } from 'react';
import './Deck.css'

class Deck extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  
  render(props) {
    return (
      <article className="deck--container">
        <h2 className="deck--name">{this.props.userDeck.deckName}</h2>
        <img
          className="deck--image"
          src="https://musingsofalifelongnerd.files.wordpress.com/2015/01/mtg-card-back.jpg"
          alt={this.props.userDeck.deckName}
        />
        <div className="price--container">
          <h3 className="price--value">${this.props.userDeck.price}</h3>
        </div>
      </article>
    );
  }
}

export default Deck;