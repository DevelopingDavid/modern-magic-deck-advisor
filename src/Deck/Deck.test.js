import React from 'react';
import Deck from './Deck';
import { shallow } from 'enzyme';

const userDeck = { deckName: 'Jund', price: 2000.00 };

describe('Deck', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Deck userDeck={userDeck} key={userDeck.deckName} />
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});