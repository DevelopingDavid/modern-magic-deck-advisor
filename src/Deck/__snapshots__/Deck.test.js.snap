// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deck should match the snapshot 1`] = `undefined`;
